cloudsync_onedrive.py

```diff
@@ -684,14 +684,9 @@ def disconnect(self):
 
     @property
     def latest_cursor(self):
-        save_cursor = self.__cursor
-        self.__cursor = self._get_url("/drives/%s/root/delta" % self._validated_namespace_id)
-        log.debug("cursor %s", self.__cursor)
-        for _ in self.events():
-            pass
-        retval = self.__cursor
-        self.__cursor = save_cursor
```

Comment:
Why don't we want to set `self.__cursor` here?

Reply:
this is a getter property, it really should not be changing state.

note that previously we were setting it for the `events()` call and then setting it back to the original value afterwards.

Reply:
that's fair, approved